### PR TITLE
create import auth resource docs

### PR DESCRIPTION
**Pull request by renebrandel**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by mauerbac — APPROVED** [review]
lgtm

File: docs/cli/auth/import.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+Run `amplify push` to complete the import procedure.
+
+## Multi-environment support
+
+When you create a new environment through `amplify env add`, Amplify CLI will assume by default that you're managing your app's Cognito resources outside of an Amplify project. You'll be asked to either import a different Cognito resource or maintain the same Cognito resource for your app's auth category.
+
+If you want to have Amplify manage your auth resources in a new environment, run `amplify remove auth` to unlink the imported Cognito resource and `amplify add auth` to create new Amplify-managed auth resources in the new environment.
+
+## Unlink an existing Cognito User Pool or Identity Pool
```
Comment (kaustavghosh06) [inline docs/cli/auth/import.md:40]: If say API or Storage is a dependency on this auth resource - will we still be able to unlink?

Comment (attilah) [reply docs/cli/auth/import.md:44]: The dependency handling logic is the very same what we have Today.

**Review by attilah — APPROVED** [review]
Beside what we synced up offline LGTM.

**Review by kaustavghosh06 — APPROVED** [review]

**Pull request merged**